build/webpack.js

```diff
@@ -207,8 +207,9 @@ export default async function getBaseWebpackConfig (dir: string, {dev = false, i
           // Because _app.js is used on every page we don't want to
           // duplicate them in other bundles.
           const chunks = module.getChunks()
+          const appBundlePath = path.normalize('bundles/pages/_app.js')
```

Comment:
Hmm, maybe moving this to `constants.js` would make more sense?

Reply:
This is fine 👍